index.html

```diff
@@ -19,6 +19,14 @@
         <!-- Comment out title for now, until we implement multiple menus -->
         <!-- <h2 class="menu-title" id="menu--site-builder">Content Editor</h2> -->
         <ul class="navigation-menu" data-once="navigation-menu" aria-labelledby="menu--site-builder">
+          <li class="menu-item level-1">
```

Comment:
@sveta1x I wouldn't make the logo part of the administration menu list. I'd make it completely independent. I would just create a `div` wrapper for now with a wrapper class + the link & logo inside. I've drafted an structure that made sense for me:

![Prototype 2023-07-03 17-04-27](https://github.com/Lullabot/navigation/assets/2225966/de65e9c2-b101-4c17-bdd9-ba29d9e90ef3)

Each `ul` should also be wrapper in a `div` plus have a heading, but it's another ticket. Just FYI :)